device-service

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
What is the purpose of this top level file?

Reply:
device services already has some ongoing ADR (provisioning for example)